app/components/l-two-columns.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div :class="b()">
+    <div :class="b('wrapper')">
+      <c-header/>
+      <div :class="b('content')">
+        <div :class="b('left')">
+          <slot name="left"></slot>
+        </div>
+        <div :class="b('main')">
+          <slot name="main"></slot>
+        </div>
+      </div>
+      <c-footer/>
+    </div>
+  </div>
+</template>
+
+<script>
+  import cHeader from '@/components/c-header';
+  import cFooter from '@/components/c-footer';
+
+  export default {
+    name: 'l-two-columns',
```

Comment:
@adrianburtscher Is this also part of this ticket?

Reply:
yes. WF-618 was combined with WF-585 after talking with Patric.